.github/workflows/CI-orchestrator.yml

```diff
@@ -31,253 +31,31 @@ jobs:
     with:
       CACHING_MODE: "CACHE_RESTORE"
 
-  check-act:
```

Comment:
Why we removed this ? was needed only for Slack ?

Reply:
I thought it was just being used for the slack message logic

Reply:
@ysibirski

Reply:
@rushby I think we should still keep it and DO NOT run `set-overall-result` stage if `needs.check-act.outputs.act == 'false'` because I do not think we want to set GitHub status when we run CI/CD locally.